Makefile

```diff
@@ -330,6 +330,33 @@ importer-image: PLATFORMS=linux/amd64
 importer-image: PUSH=--load
 importer-image: importer-image-build
 
+
+# Build the kueue-viz dashboard images (frontend and backend)
+.PHONY: kueue-viz-image-build
+kueue-viz-image-build:
+	@for component in backend frontend; do \
+		$(IMAGE_BUILD_CMD) \
+			-t $(IMAGE_REGISTRY)/kueue-viz-$$component:$(GIT_TAG) \
+			-t $(IMAGE_REGISTRY)/kueue-viz-$$component:$(RELEASE_BRANCH)-latest \
+			--platform=$(PLATFORMS) \
+			--build-arg BASE_IMAGE=$(BASE_IMAGE) \
+			--build-arg BUILDER_IMAGE=$(BUILDER_IMAGE) \
```

Comment:
I think it will not work with frontend Dockerfile.

Reply:
on my local build, docker accepts the arguments. It is just that it is ignore for the frontend build. 

I want to align the 2 images in the future to have distroless images only. The frontend image is a bit a struggle right now, because of the requirement to passe an env var containing the backed url. And so, it uses `npx` which is not available in any distroless image that I found.

So, we have 2 choices here:
- align the 2 builds like I did here, having a for loop with the 2 components but passing 2 ignored build-arg
- duplicate the code and remove the build args that will be ignore anyway

I am ok with both, just let me know which one you prefer :-) ?

Reply:
Actually, it is not a complete duplication because the configuration differs. I would suggest splitting it into separate targets.

@mimowo WDYT?

Reply:
changing that to 2 separate statements then